.github/workflows/package-Preview.yml

```diff
@@ -46,5 +46,4 @@ jobs:
 
     - name: Publish Nuget
       run: |
-        nuget setApiKey ${{ secrets.NUGET_TOKEN }} -Source https://api.nuget.org/v3/index.json
-        nuget push **/${{env.PROJECT}}/bin/**/*.nupkg -Source https://api.nuget.org/v3/index.json
\ No newline at end of file
+        nuget push **/${{env.PROJECT}}/bin/**/*.nupkg -Source https://api.nuget.org/v3/index.json -ApiKey ${{ secrets.NUGET_TOKEN }} -SymbolApiKey ${{ secrets.NUGET_TOKEN }}
```

Comment:
@srprash FYI. This may fix the symbols error you also obseverved when pushing to nuget

Reply:
It worked so I applied to the other workflows: #131